packages/tc-schema-sdk/lib/updater.js

```diff
@@ -3,6 +3,14 @@ const fetch = require('node-fetch');
 const { getSchemaFilename } = require('@financial-times/tc-schema-file-name');
 const readYaml = require('./read-yaml');
 const { version: libVersion } = require('../../../package.json');
+const { compatBackward } = require('./compat-backward');
+
+// e.g HasChild -> HAS_CHILD
+const toSnakeUpperCase = name =>
+	name
+		.replace(/([A-Z])/g, '_$1')
+		.replace(/^_|_$/g, '')
+		.toUpperCase();
```

Comment:
This function will generate a relationship name automatically from it's rel name